src/index.html

```diff
@@ -0,0 +1,97 @@
+          <svg height="32" width="32" class="icon" viewBox="0 0 12 16" version="1.1">
+            <path fill-rule="evenodd" d="M11.41 9H.59C0 9 0 8.59 0 8c0-.59 0-1 .59-1H11.4c.59 0 .59.41.59 1 0 .59 0 1-.59 1h.01zm0-4H.59C0 5 0 4.59 0 4c0-.59 0-1 .59-1H11.4c.59 0 .59.41.59 1 0 .59 0 1-.59 1h.01zM.59 11H11.4c.59 0 .59.41.59 1 0 .59 0 1-.59 1H.59C0 13 0 12.59 0 12c0-.59 0-1 .59-1z"></path>
+          </svg>
+        </button>
+        <a class="navbar-brand" href="#page-top">
+          <svg viewBox="0 0 10 10" class="logo">
+            <g>
+              <path d="m1,1h4v8h-4v-2M9,3v-2h-4v8h4v-2" />
+            </g>
+          </svg>
+          <span>Jerahmeel Cosinas</span>
+        </a>
+      </div>
+
+      <div class="collapse navbar-collapse navbar-right navbar-main-collapse">
+        <ul class="nav navbar-nav">
+          <li class="hidden">
+            <a href="#page-top"></a>
+          </li>
+          <li class="page-scroll">
+            <a href="#about">About</a>
+          </li>
+          <li class="page-scroll">
+            <a href="#projects">Projects</a>
+          </li>
+          <li class="page-scroll">
+            <a href="#contact">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <section class="intro content-section">
+    <div class="container">
+      <h1 class="brand-heading"><span>Jerahmeel Cosinas</span></h1>
+      <p class="tagline">Programmer · Curious</p>
+      <p class="description"></p>
+      <div class="page-scroll">
+        <a href="#about" class="btn btn-circle">
+          <i class="fa fa-angle-double-down animated"></i>
+        </a>
+      </div>
+    </div>
+  </section>
+
+  <section id="about" class="container content-section">
+    <h2>Jerahmeel</h2>
+    <p>Donec eu ornare tellus. Fusce fringilla tortor velit, sit amet faucibus nisl convallis et. In tellus neque, blandit ut rhoncus ultricies, porta ac sem. Duis fringilla ac augue eu hendrerit. Quisque sem lorem, consectetur sed rutrum vel, scelerisque eu arcu. Donec eleifend massa eget tempus consectetur.</p>
+    <p>Proin ut risus at lacus iaculis facilisis. Maecenas justo purus, commodo eu metus ut, semper accumsan est. Suspendisse maximus facilisis lorem vitae scelerisque. Nunc suscipit, turpis ac lacinia vulputate, enim risus congue sem, ut semper urna purus ut ex.</p>
+  </section>
+
+  <section id="projects" class="content-section">
+    <div class="container">
+      <h2>Projects</h2>
+      WIP
+    </div>
+  </section>
+
+  <section id="contact" class="content-section">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-8 col-lg-offset-2">
+          <h2>Contact</h2>
+          WIP
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+  <script src="//netdna.bootstrapcdn.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
```

Comment:
We can’t put these in the head?

Reply:
this is javascript, standard practice is, javascript goes at the bottom of the page. this is to:

1. ensure that the body has been parsed
1. not block parsing of the DOM

that said, this might not be a problem in http2 anymore, I'm not sure.